cellprofiler/gui/figure.py

```diff
@@ -1588,19 +1588,29 @@ def subplot_imshow_labels(
                      consistent label colors in multiple displays)
         :return:
         """
-        if background_image is None:
-            background_image = numpy.zeros_like(image, dtype=numpy.float32)
-            opacity = 1.0
-        else:
+        if background_image is not None:
             opacity = 0.7
-
-        label_image = cellprofiler.object.overlay_labels(
-            labels=image,
-            opacity=opacity,
-            pixel_data=background_image,
-            max_label=max_label,
-            seed=seed
-        )
+            label_image = cellprofiler.object.overlay_labels(
+                labels=image,
+                opacity=opacity,
+                pixel_data=background_image,
+                max_label=max_label,
+                seed=seed
+            )
+            colormap = None
+        else:
+            # Mask the original labels
+            label_image = numpy.ma.masked_where(image == 0, image)
+            # Create a custom colormap
+            # Based off of the prism color map (with higher frequency):
+            # https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/_cm.py#L53-L57
+            new_prism = {
+                'red': lambda x: 0.75 * numpy.sin((x * 42 + 0.25) * numpy.pi) + 0.67,
```

Comment:
A minor detail, but `0.67`?

Reply:
I literally copied this block straight from matplotlib's source (linked above) and adjusted the frequency `20.9 --> 42` to make it a bit higher. What would you suggest changing this to?